cli/azd/test/functional/cli_test.go

```diff
@@ -81,9 +81,9 @@ type cliConfig struct {
 
 func (c *cliConfig) init() {
 	c.CI = os.Getenv("CI") != ""
-	c.ClientID = os.Getenv("AZD_TEST_CLIENT_ID")
-	c.ClientSecret = os.Getenv("AZD_TEST_CLIENT_SECRET")
-	c.TenantID = os.Getenv("AZD_TEST_TENANT_ID")
+	// c.ClientID = os.Getenv("AZD_TEST_CLIENT_ID")
+	// c.ClientSecret = os.Getenv("AZD_TEST_CLIENT_SECRET")
+	// c.TenantID = os.Getenv("AZD_TEST_TENANT_ID")
```

Comment:
I think the main purpose of this test is to check azd work with client-secret.  If we change this, we would be loosing coverage for it.
Can we remove/deprecate support for client-secret?   Otherwise we might want coverage

Reply:
Trying to get the builds off the floor first (with minimal damage) and then will revisit the coverage we want here

It's possible we'll just lose coverage in CI and have to move this to be a manual test if we get to a place where we can't be using client secrets in CI.

The place this was ready was `t.Skip`'d for now.